packages/ebayui-core/src/components/ebay-snackbar-dialog/component.ts

```diff
@@ -5,7 +5,7 @@ import type { Input as BaseInput } from "../components/ebay-dialog-base/componen
 interface SnackbarDialogInput extends Omit<BaseInput, `on${string}`> {
     layout?: "row" | "column";
     action?: BaseInput["action"] & {
-        accessKey?: string;
+        accesskey?: string;
```

Comment:
Will this be a breaking change for TS users? Maybe we should support both, and derive `accesskey` from `accessKey`

Reply:
I believe since this was not working at all, we should change it to the right attribute.